data/human/human missions.txt

```diff
@@ -6891,3 +6891,370 @@ mission "Returning to Paradise"
+				`	(Charge him with mutiny.)`
+					goto blaster
+				`	(Let him go to the cargo bay and lock down the drones.)`
+					goto escape
+
+			label "call drones"
+			`	"Aye aye, Captain," he says as he hits a button. "Drone ETA: two minutes, fifteen seconds."`
+			`	"Get to the cargo bay," you say. "Lock down the drones as soon as they arrive and get belted in. We'll take off on your mark."`
+
+			label escape
+			`	Less than a minute later, you hear Bunker on the comms. "Drones are locked down and ready to go."`
+			`	The moment you pull out of the canyon, pirate Quicksilvers appear from every direction. You shunt the <ship>'s repulsors into full blast and accelerate into space with the pirates in hot pursuit.`
+				launch
+
+			label blaster
+			`	You get up and point your pistol at Bunker while ordering him to stand down. "Don't do this," he says, shrinking from you as he puts his hands up. As you are restraining him, you notice the humming of thrusters from outside - then the <ship> receives a hail of particle cannon fire out of nowhere! You turn around and see pirate Quicksilvers all converging towards you, whittling the <ship>'s shields down in a split second. You shunt the <ship>'s repulsors to full blast to get into orbit, but a volley of particle fire breaks through and takes the repulsors offline. The last thing you experience is the ship slowing to a halt, pausing in the air for just a moment before the pilot seat goes out underneath you, the cockpit filled with screams as the <ship> makes a one way trip towards the surface.`
+				die
+
+	npc
+		personality heroic nemesis vindictive launching
+		government "Pirate"
+		ship "Quicksilver" "Dog's Leash"
+		ship "Quicksilver" "Burning Whip"
+		ship "Quicksilver" "Iron Chains"
+
+	on visit
+		dialog `Your escort carrying the spy drones was left behind. Better take off and wait for them to enter the system.`
+
+
+
+mission "Adelita 5"
+	name "Return Adelita to <planet>"
+	description "Bring everyone back to <destination>. (You will be paid <payment> after you complete the entire job.)"
+	landing
+	passengers 3
+	cargo "stealth drones" 1
+	"apparent payment" 200000
+	source "Maker"
+	destination "Moonshake"
+	to offer
+		has "Adelita 4: done"
+
+	on offer
+		conversation
+			`You managed to deal with the Quicksilvers and arrive on <origin> alive. As soon as you land, Ale and River rush on board, obviously relieved to be out of their disguises, and they spend a lot of time talking with Bunker about his part of the mission. Although exhausted, all of them are eager to get back and process what they filmed - both emotionally and literally.`
+			`	"That was horrible, but we got it done," Ale mumbles. "Can we leave now?"`
+				accept
+
+	on visit
+		dialog phrase "generic cargo and passenger on visit"
+
+	on complete
+		payment 220000
+		log `Helped the underground journalism collective "Adelita Zero" gather video evidence of Syndicate participation in the slave trade while escaping from pirates.`
```

Comment:
```suggestion
		log `Helped the underground journalism collective "Adelita" gather video evidence of Syndicate participation in the slave trade while escaping from pirates.`
```
I assumed that Adelita Zero = Alejandra? The collective is previously only referred to as the "Adelita Collective" or "Adelita."

Reply:
I think there may be more context about why the "Zero" is there in the later missions, I'll check.

Reply:
Doesn't look like it. I think they only call themselves Adelita Zero in the note.